meson_options.txt

```diff
@@ -183,7 +183,9 @@ option('pamconfdir', type : 'string',
 option('docdir', type : 'string',
        description : 'documentation directory')
 option('install-sysconfdir', type : 'boolean', value : true,
-       description : 'install configuration files to $sysconfdir')
+       description : 'install configuration files and directories to $sysconfdir')
+option('install-sysconfdir-samples', type : 'boolean', value : true,
+       description : 'install sample configuration files (commented defaults) to $sysconfdir')
```

Comment:
mmh perhaps it would look better if the original was a combo switch, rather than having a new option? eg:

```
option('install-sysconfdir', type : 'combo', choices : ['true', 'no-samples', 'false'],
```

```
install_sysconfdir = get_option('install-sysconfdir') != 'false' 
install_sysconfdir_samples = get_option('install-sysconfdir') == 'true'
```

Reply:
That's a good idea! I've updated the PR.